src/screens/TopicMemberScreen/index.js

```diff
@@ -88,7 +88,7 @@ const TopicMemberScreen = () => {
     return () => {
       scrollY.removeListener(listener);
     };
-  }, []);
+  }, [scrollY]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dependency array of the `useEffect` hook has been updated to include `scrollY`. This change is good if `scrollY` is expected to change over the component lifecycle and you want to add/remove listeners accordingly. However, if `scrollY` does not change, this could lead to unnecessary listener removals and additions. Please ensure that this change aligns with your intended behavior.
```diff
-  }, []);
+  }, [scrollY]);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->